### DISCUSSION TITLE
Is there a templates marketplace and a way to submit those to Webstudio?

### DISCUSSION THREAD

**Discussion by schemetastic** [webstudio-is/webstudio-community#185] [Q&A] [answered]
I've found in [Roadmap](https://github.com/orgs/webstudio-is/projects/11) that is something already done... but I haven't found any in the official website, and also I haven't found any in the editor aside from the few templates already there.

It would be nice to be able to submit your designs and be able to monetize them through Lemon Squeezy or Gumroad and to have a marketplace to direct some traffic to it.

I believe that is a great way to make Webstudio grow, because designers like to share their work in social media and that could open a new market for designer. Also, having a marketplace is a plus if you have clients and want to show them in advance designs that they could use.

**Comment by kof** [comment, marked as answer]
https://docs.webstudio.is/university/marketplace

**Reply by schemetastic** [reply]
I see... the Marketplace is a recent feature and still needs support, I'll be glad to contribute to it in a near future 🫡

**Reply by kof** [reply]
not sure what you mean, its available for a while, and you can submit projects with a few clicks ... its all there

**Reply by schemetastic** [reply]
Excuse me, I'm new with Web Studio and I couldn't find that information, specially because I thought I was going to find it in a very visible space in the front page (the marketplace). In the editor, I entered to the marketplace section and there are very few templates (Like 5 I think), that's why I thought it was recent, and it needed support from designers to grow it, and I've got some interest in it because I am developing Framer templates and I thought this could be an opportunity to reach another audience 🤷‍♂️

**Reply by kof** [reply]
yes, that's exactly the case

**Reply by schemetastic** [reply]
Great, I'll try to set some time aside to develop a template in a near future, I'm still learning to use the editor. 👌
